lib/vmdb/settings/validator.rb

```diff
@@ -31,6 +31,23 @@ def validate
 
       private
 
+      def is_integer_with_method?(num)
```

Comment:
@Fryguy As you can see, this got a bit... complicated.

I was starting with the tests I had below, and `"Five minutes"` was a valid value, even though it would return `0`.  This method was the clearest way I could think of doing it.

Didn't test it directly, since the followup tests cover most of the cases I can think of, but I can add some if you think they are needed.

Reply:
Weird, is that something we want to support or a bug with the existing `to_i_with_method`? It should result in `5` I believe.

Reply:
@jrafanie I would argue it should be `nil`, since we aren't doing natural language parsing with that method.  The `.to_i_with_method` is just supporting stuff like `"5.minutes"` in string form (for yaml files), and not `"Five minutes"` for example.

Reply:
We have a method called `number_with_method?` that I think does most if not all of this checking.